index.js

```diff
@@ -481,3 +495,33 @@ function cuuid () {
   const str = (Date.now().toString(16) + Math.random().toString(16).slice(2) + Math.random().toString(16).slice(2) + Math.random().toString(16).slice(2)).slice(0, 32)
```

Comment:
wow

Reply:
this function is a few hundreds of lines shorter the uuid package.

Reply:
😂